src/libs/API.js

```diff
@@ -582,7 +582,7 @@ function Wallet_GetOnfidoSDKToken(firstName, lastName, dob) {
  * @returns {Promise}
  */
 function Plaid_GetLinkToken() {
-    return Network.post('Plaid_GetLinkToken', getPlaidLinkTokenParameters(), CONST.NETWORK.METHOD.POST, true);
+    return Network.post('Plaid_GetLinkToken', getPlaidLinkTokenParameters(), CONST.NETWORK.METHOD.POST, false);
```

Comment:
The `shouldUseSecure` param is false by default, so you can just remove that parameter.

Reply:
I think something is not working here, the code is actually like so `return Network.post('Plaid_GetLinkToken', getPlaidLinkTokenParameters(), CONST.NETWORK.METHOD.POST, true);`